Makefile

```diff
@@ -141,27 +139,25 @@ endif
 
 ifeq ($(UNICORN_SHARED),yes)
 ifneq ($(filter MINGW%,$(UNAME_S)),)
-LIBRARY = $(BLDIR)/$(LIBNAME).$(EXT)
+LIBRARY = $(LIBNAME).$(EXT)
 else ifneq ($(filter CYGWIN%,$(UNAME_S)),)
-LIBRARY = $(BLDIR)/cyg$(LIBNAME).$(EXT)
```

Comment:
looks like the `cyg` prefix is good to differentiate between cygwin and mingw. i'll change the makefile back and add to python